lib/manager/gradle/index.js

```diff
@@ -0,0 +1,49 @@
+const { execSync } = require('child_process');
+const fs = require('fs');
+
+const gradle = require('./build-gradle');
+
+function extractDependencies(content, fileName, config) {
+  logger.debug('gradle.extractDependencies()');
+  const deps = [{ name: 'dummy' }];
+  const gradleFile = `${config.localDir}/${fileName}`;
+
+  fs.writeFileSync(gradleFile, content);
+  execSync('gradle dependencyUpdates -Drevision=release', {
+    stdio: 'inherit',
+    cwd: config.localDir,
+  });
```

Comment:
Is it necessary to run this command here, or could it be skipped? It looks like you are returning a dummy deps no matter what.

Reply:
this command generates the file with the dependencies report that then is used in the getPackageUpdates function. 

I'm not returning the dependencies here because I didn't find it necessary. But I should do it, at least for the sake of consistency